### PR TITLE
Routes to Success Page

### PR DESCRIPTION
**Pull request by angepol**
- Routes to Success Page
- Adds WIP error handling for paymentIntent

### CODE REVIEW

File: cha-ching-app/src/CheckoutForm.js [hunk 1]
```diff
@@ -1,36 +1,42 @@
...
+      .confirmPayment({
+        elements,
+        confirmParams: {
+          return_url: "http://localhost:3000/success",
+        },
+      })
+      .then((data) => console.log("What am I (data):", data))
+      .catch((err) =>
+        console.log("Something went wrong confirming the payment:", err),
+      );
```
Comment (Ashlynn223) [inline cha-ching-app/src/CheckoutForm.js:22]: Stripe ends up handling errors that happen as a result of confirmPayment so it ends up printing out any form error in the `.then` block. Could we check that when we pass an undefined value inside the confirmPayment that it hits this catch block?

File: cha-ching-app/src/CheckoutForm.js [hunk 2]
```diff
@@ -1,36 +1,42 @@
...
-        </div>
+  return (
+    <div>
+      <div className="header">Cha-ching</div>
+      <div className="paymentFormContainer">
+        <form id="payment-form">
+          <div className="paymentElementcontainer">
+            <PaymentElement />
+          </div>
+        </form>
```
Comment (Ashlynn223) [inline cha-ching-app/src/CheckoutForm.js:33]: This form might not be needed, in the tutorials we saw it was being used to associate the button that's inside the form to the payment element but we ended up bringing that button outside the form. Lets try removing it and see if it behaves the same

File: cha-ching-app/src/Error.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+const Error = (message, code, type) => {
+  return (
+    <div className="errorWrapper">
+      <div className="errorContainer">
+        <div className="errorText">error</div>
+      </div>
+    </div>
+  );
+};
```
Comment (Ashlynn223) [inline cha-ching-app/src/Error.js:11]: Nice! It's okay to make this a basic component for now that is not yet reusable (meaning removing the params that are getting passed in). Later on we could make this a more dynamic component that can work for multiple cases.

File: cha-ching-app/src/error.css [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+  margin-right: auto;
+  position: absolute;
+  right: 0;
+  text-align: center;
+  top: 100px;
+  width: 326px;
+}
+
+.errorWrapper {
+  display: flex;
```
Comment (Ashlynn223) [inline cha-ching-app/src/error.css:31]: Thought this was a cool article with different ways of showing visuals for errors in the app: https://blog.hubspot.com/blog/tabid/6307/bid/33766/10-clever-website-error-messages-from-creative-companies.aspx

File: server/app.js [hunk 5]
```diff
@@ -13,16 +13,13 @@ app.get("/", (req, res) => {
...
-
 app.get("/payment-intent", async (req, res) => {
   const paymentIntent = await stripe.paymentIntents.create({
     amount: 2000,
     currency: "usd",
   });
-  res.json({ clientSecret: paymentIntent.client_secret });
+  // res.json({ clientSecret: paymentIntent.client_secret });
+  res.json({ clientSecret: undefined });
 });
```
Comment (Ashlynn223) [inline server/app.js:23]: We can swap this back to the real clientSecret 👀

File: cha-ching-app/src/Error.js (hunk 3 above)
Comment (angepol) [reply cha-ching-app/src/Error.js:11]: Sounds good! I will remove them for now. Thank you!

File: cha-ching-app/src/error.css (hunk 4 above)
Comment (angepol) [reply cha-ching-app/src/error.css:31]: Great, i'll look into this for some stying inspiration.

**Pull request merged**